src/Farmer/Deploy.fs

```diff
@@ -117,13 +117,13 @@ module Az =
     // The what-if operation doesn't make any changes to existing resources. Instead, it predicts the changes if the specified template is deployed.
     let whatIf resourceGroup deploymentName templateFilename parameters = deployOrValidate WhatIf resourceGroup deploymentName templateFilename parameters
     /// Generic function for ZipDeploy using custom command (based on application type)
-    let private zipDeploy command appName getZipPath resourceGroup =
-        let packageFilename = getZipPath deployFolder
+    let internal zipDeploy az command appName getZipPath resourceGroup =
```

Comment:
Is this extra parameter to allow "mocking" or injecting the actual `az` command? That's an interesting approach :-) I think it might be better to simply have the `zipDeploy` function return a string and have the callers pipe it into `az`.